plane/source/docs/takeoff-landingpage.rst

```diff
@@ -0,0 +1,55 @@
+.. _takeoff-landingpage:
+
+=======
+Takeoff
+=======
+
+Taking the plane off can be accomplished in many ways: Rolling Takeoff, Hand Launch, Bungee, or VTOL (Verical Takeoff and Landing) if a :ref:`quadplane-support`. And it can be done manually or automatically.
+
+Rolling Takeoff
+===============
+
+In this case, the plane has landing gear with steerable nose gear, usually coupled to the rudder control and the plane is usually launched from a smooth grass or paved surface. Once aligned nose into the wind, throttle is gradually increased to takeoff value (usually full throttle) while the aircraft direction is controlled by rudder. Once sufficient airspeed to lift off is obtained, slight up elevator will rotate the nose to climb attitude.
+
+MANUAL or FBWA modes are most commonly used during takeoffs.
```

Comment:
Add that for FBWA or other automatic modes, the STER parameters may need to be tuned if the nosewheel is configured as a steering servo.

Reply:
good point will add